.travis.yml

```diff
@@ -1,39 +1,43 @@
-language: julia
+# For learning how to customize the CI check out
+# Continous Integration:
+# https://docs.travis-ci.com/user/languages/julia
+# https://github.com/marketplace/travis-ci
+# Code Coverage:
+# https://github.com/JuliaCI/Coverage.jl
+# Documentation:
+# https://github.com/JuliaDocs/Documenter.jl
```

Comment:
Add some whitespace between sections or use hypen to make URLs into lists. Currently this is a little hard to read